PhysicsTools/PatAlgos/plugins/PATMuonMerger.cc

```diff
@@ -33,24 +38,38 @@ class PatMuonMerger : public edm::stream::EDProducer<> {
   edm::EDGetTokenT<std::vector<pat::Muon>> muonToken_;
   edm::EDGetTokenT<std::vector<pat::PackedCandidate>> pfCandToken_;
   edm::EDGetTokenT<std::vector<pat::PackedCandidate>> lostTrackToken_;
+  
+
 };
 
-PatMuonMerger::PatMuonMerger(const edm::ParameterSet& iConfig)
+PATMuonMerger::PATMuonMerger(const edm::ParameterSet& iConfig)
     : muons_(iConfig.getParameter<edm::InputTag>("muons")),
-      muonsCut_(iConfig.existsAs<std::string>("muonCut") ? iConfig.getParameter<std::string>("muonCut") : ""),
+      muonsCut_(iConfig.getParameter<std::string>("muonCut")),
       pfCandidate_(iConfig.getParameter<edm::InputTag>("pfCandidates")),
-      pfCandidateCut_(
-          iConfig.existsAs<std::string>("pfCandidatesCut") ? iConfig.getParameter<std::string>("pfCandidatesCut") : ""),
+      pfCandidateCut_(iConfig.getParameter<std::string>("pfCandidatesCut")),
       lostTrack_(iConfig.getParameter<edm::InputTag>("otherTracks")),
-      lostTrackCut_(iConfig.existsAs<std::string>("lostTrackCut") ? iConfig.getParameter<std::string>("lostTrackCut")
-                                                                  : "") {
+      lostTrackCut_(iConfig.getParameter<std::string>("lostTrackCut")) {
   muonToken_ = consumes<std::vector<pat::Muon>>(muons_);
   pfCandToken_ = consumes<std::vector<pat::PackedCandidate>>(pfCandidate_);
   lostTrackToken_ = consumes<std::vector<pat::PackedCandidate>>(lostTrack_);
   produces<std::vector<pat::Muon>>();
 }
 
-void PatMuonMerger::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
+
+void
+PATMuonMerger::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // mergedMuons
+  edm::ParameterSetDescription desc;
+  desc.add<std::string>("muonCut", "pt>15 && abs(eta)<2.4");
```

Comment:
In the previous version the default value (in case the "muonCut" parameter was missing in the explicit config) was an empty string 
The same was for the "pfCandidatesCut"

Perhaps, keeping the "no cuts" config as the default one, as it was before, could be more appropriate. But I have no strong opinion on it.